store/sqlstore/upgrade.go

```diff
@@ -1301,23 +1301,37 @@ func fixCRTChannelMembershipCounts(sqlStore *SqlStore) {
 func upgradeDatabaseToVersion600(sqlStore *SqlStore) {
 	// if shouldPerformUpgrade(sqlStore, Version5380, Version600) {
 
+	if sqlStore.DriverName() == model.DatabaseDriverMysql {
+		if sqlStore.DoesColumnExist("Posts", "ParentId") {
+			sqlStore.GetMaster().ExecNoTimeout("ALTER TABLE Posts MODIFY COLUMN FileIds text, MODIFY COLUMN Props JSON, DROP COLUMN ParentId")
```

Comment:
This excludes the column exist check for FileIds and Props. Although it is highly unlikely a posts table would exist without those columns.

Reply:
Yes I am assuming those exist otherwise the instance is broken to start with.